openlibrary/macros/AvailabilityButton.html

```diff
@@ -66,3 +66,7 @@
   <form>
     <input type="button" class="cta-btn--missing cta-btn" disabled value="No ebook available">
   </form>
+  $if page.ia:
+    $ daisy_url = "/ia/%s/daisy" % page.ia[0]
+    $:macros.daisy(page, url=daisy_url, msg="Print-disabled access available")
```

Comment:
What about internationalisation for the availability messaging? These should be wrapped in an appropriate i18n display wrapper